bundles/framework/layerselector2/resources/locale/en.js

```diff
@@ -1,93 +1,94 @@
-            "minChars": "The search term is too short. Type at least four characters."
-        },
-        "loading": "Searching...",
-        "filter": {
-            "text": "Search map layers.",
-            "keywordsTitle": "Keywords",
-            "shortDescription": "Search map layers by map layer name, data producer name or keyword.",
-            "description": "Search map layers by map layer name, data producer name or keyword. You can type a whole term or a part of it. The search term must be at least four characters long.",
-            "inspire": "By Theme",
-            "organization": "By Data Provider",
-            "published": "Users",
-            "didYouMean": "Did you mean:"
-        },
-        "published": {
-            "organization": "Published map layer",
-            "inspire": "Published map layer"
-        },
-        "tooltip": {
-            "type-base": "Background map",
-            "type-wms": "Map layer (WMS, WMTS)",
-            "type-wfs": "Data product (WFS)",
-            "type-wfs-manual": "Refresh feature data by clicking 'Feature Data' or 'Refresh' button in the map window.",
-            "type-timeseries": "Time series layer",
-            "unsupported-srs": "Unsupported map projection",
-            "addLayer": "Add map layer"
-        },
-        "backendStatus": {
-            "OK": {
-                "tooltip": "The map layer is currently available.",
-                "iconClass": "backendstatus-ok"
+    {
+        'lang': 'en',
+        'key': 'LayerSelector',
+        'value': {
+            'title': 'Map Layers',
+            'desc': '',
+            'errors': {
+                'title': 'Error',
+                'generic': 'The system error occurred.',
+                'loadFailed': 'The map layers could not be loaded. Reload the webpage in your browser and select map layers again.',
+                'noResults': 'No search results could be found. Please check the search term.',
+                'noResultsForKeyword': 'No map layers could be found. Please check the search term.',
+                'minChars': 'The search term is too short. Type at least four characters.'
             },
-            "DOWN": {
-                "tooltip": "The map layer is not currently available.",
-                "iconClass": "backendstatus-down"
+            'loading': 'Searching...',
+            'filter': {
+                'text': 'Search map layers.',
+                'keywordsTitle': 'Keywords',
+                'shortDescription': 'Search map layers by map layer name, data producer name or keyword.',
+                'description': 'Search map layers by map layer name, data producer name or keyword. You can type a whole term or a part of it. The search term must be at least four characters long.',
+                'inspire': 'By Theme',
+                'organization': 'By Data Provider',
+                'selected': 'Selected layers',
```

Comment:
ESLINT was screaming red on this file, but this localization was added

Reply:
All changes to localization files should be made under layerlist/resources/locale. Can't say if there are any real changes here.